src/generators/css-filter.ts

```diff
@@ -63,7 +77,16 @@ export function cssFilterStyleheetTemplate(filterValue: string, reverseFilterVal
     });
 
     if (!frameURL) {
-        const [r, g, b] = applyColorMatrix([255, 255, 255], createFilterMatrix(config));
+        let r: number;
+        let g: number;
+        let b: number;
+        if (getChromeVersion() == null){
```

Comment:
The same thing is called twice (in `if` and `else` blocks). Make it called once. Also, only the array is different in all 3 blocks. So we can shorten it like `const [r, g, b] = hasIssue() ? [x] : [y];`

Reply:
I don't get what you try to mean